adafruit_tinylora/adafruit_tinylora.py

```diff
@@ -235,16 +218,14 @@ def send_data(self, data, data_length, frame_counter, timeout=2):
         # set length of LoRa packet
         lora_pkt_len = 9
         # load encrypted data into lora_pkt
-        for i in range(0, data_length):
-            lora_pkt[lora_pkt_len + i] = enc_data[i]
+        lora_pkt[lora_pkt_len:data_length] = enc_data[0:data_length]
```

Comment:
fix 
 lora_pkt[lora_pkt_len:lora_pkt_len+data_length] = enc_data[0:data_length]

Reply:
oops -- just realized my feather_m0 test was using older version -- updated to last commit - and it does work -- seeing this, I'm not sure why...